tests/src/test/scala/cats/tests/ReaderWriterStateTTests.scala

```diff
@@ -344,6 +344,13 @@ class ReaderWriterStateTTests extends CatsSuite {
     checkAll("ReaderWriterStateT[ListWrapper, String, Int, String, Int]",
       SemigroupKTests[ReaderWriterStateT[ListWrapper, String, Int, String, ?]].semigroupK[Int])
   }
+
+  {
+    implicit def F = ListWrapper.monad
```

Comment:
copy/pasted it from some code above, so it's not the only place where it should be fixed :)